xLights/models/Model.cpp

```diff
@@ -4225,13 +4225,13 @@ void Model::InitRenderBufferNodes(const std::string& tp, const std::string& came
         logger_base.warn("Model::InitRenderBufferNodes BufferWi was 0 ... overridden to be 1.");
         bufferWi = 1;
     }
-    if (bufferHt > 10000) {
-        logger_base.warn("Model::InitRenderBufferNodes BufferHt was overly large ... overridden to be 10000.");
-        bufferHt = 10000;
+    if (bufferHt > 100000) {
+        logger_base.warn("Model::InitRenderBufferNodes BufferHt was overly large ... overridden to be 100000.");
+        bufferHt = 100000;
     }
-    if (bufferWi > 10000) {
-        logger_base.warn("Model::InitRenderBufferNodes BufferWi was overly large ... overridden to be 10000.");
-        bufferWi = 10000;
+    if (bufferWi > 100000) {
+        logger_base.warn("Model::InitRenderBufferNodes BufferWi was overly large ... overridden to be 100000.");
+        bufferWi = 100000;
```

Comment:
Kind of wondering if these checks should only be applied if (bufferWi*bufferHi) > 2.1M or so.  Basically, a really long single line is fine, but kept to some level of reasonableness.  (~2.1M would allow a 1080p hd virtual matriix which would be kind of a lot for rendering.).

Reply:
Like a wrap the above in another if -- seems like a good idea. Just trying to stop that crash from wacky values.